### PR TITLE
Update/nrfdl log dev

### PR DESCRIPTION
**Pull request by nicolai-nordic**
- Only propgate nrfdl log messages if in development (i.e. app is not packaged).
- Update nrfdl log message formatting slightly, i.e. add log-level.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.16.3 - 2022-01-28
+
+### Changed
+
+-   Only propagate device-lib logs for production builds.
```
Comment (datenreisender) [inline Changelog.md:13]: The inverse is true, right? 
```suggestion
-   Only propagate device-lib logs while in development.
```

File: package.json [hunk 2]
```diff
@@ -147,4 +147,4 @@
         "extends": "./config/eslintrc.json"
     },
     "prettier": "./config/prettier.config.js"
-}
+}
```
Comment (datenreisender) [inline package.json:150]: Please add the final newline back.

File: src/Device/deviceLister.ts [hunk 3]
```diff
@@ -30,6 +31,7 @@ nrfDeviceLib.setTimeoutConfig(deviceLibContext, { enumerateMs: 3 * 60 * 1000 });
 export const getDeviceLibContext = () => deviceLibContext;
 
 export const logNrfdlLogs = (evt: LogEvent) => {
+    if (remote.app.isPackaged) return;
```
Comment (datenreisender) [inline src/Device/deviceLister.ts:34]: Nice, I did not know about `isPackaged`. Maybe we can use this also in other places instead of NODE_ENV.

**Review by datenreisender — COMMENTED** [review]
With this changes in place, the logs from device-lib are _never_ propagated, and thus the “Verbose logging” does not work anymore, right? Is that intended?

**Comment by nicolai-nordic** [comment]
> With this changes in place, the logs from device-lib are _never_ propagated, and thus the “Verbose logging” does not work anymore, right? Is that intended?

You are right, good catch, this is not intended behavior.

**Review by jonasem — APPROVED** [review]

**Pull request merged**